index.html

```diff
@@ -1,21 +1,144 @@
 <!DOCTYPE html>
 <html xmlns="http://www.w3.org/1999/xhtml">
 <head>
-  <title>onedrive explorer</title>
+  <title>OneDrive Browser Sample</title>
   <link rel="stylesheet" type="text/css" href="style.css"/>
   <script type="text/javascript" src="https://code.jquery.com/jquery-1.11.1.min.js"></script>
-  <script id="odauth" src="odauth.js"
-   clientId="0000000048144E4F" scopes="onedrive.readonly wl.signin"
-   redirectUri="https://onedrive.github.io/odx/callback.html"></script>
+  <script id="odauth" src="odauth.js"></script>
+  <link rel="icon" href="https://d.sfx-df.ms/images/favicon.ico" type="image/x-icon">
+  <link rel="mask-icon" href="https://d.sfx-df.ms/images/mask_icon.svg" color="#094AB2">
+  <script>
+    function signInToOneDrive()
+    {
+      // Register your own application at https://apps.dev.microsoft.com
+      // and set the "clientId" and "redirectUri" variables accordingly.
+      var appInfo = {
+        "clientId": "e9a57c07-69e4-41f5-b868-0f123d2fda17",
+        "redirectUri": "http://localhost:9999/callback.html",
```

Comment:
The live version uses a copy of this code, and does't pull from the repo. It has the right values for the live version.